Source/WebKit/UIProcess/Cocoa/ProcessAssertionCocoa.mm

```diff
@@ -372,7 +372,7 @@ static ASCIILiteral runningBoardDomainForAssertionType(ProcessAssertionType asse
     , m_reason(reason)
 {
 #if USE(EXTENSIONKIT)
-    if (process.extensionProcess()) {
+    if (process.extensionProcess().index() != std::variant_npos) {
```

Comment:
May be nicer to use `valueless_by_exception()`

Reply:
Actually, see my comment above about calling `ExtensionProcess()` I do not believe this does anything at the moment?